seed/tests/test_project_views.py

```diff
@@ -822,7 +822,7 @@ def test_move_inventory(self):
         )['project']
         url = "{}?organization_id={}&inventory_type=property".format(
             reverse_lazy(
-                'apiv2:projects-move', args=[project['slug']]
+                'api:v2:projects-move', args=[project['slug']]
```

Comment:
It would be really nice if Github's diff view would accept a query parameter so I could filter out certain diffs.  I can already filter whitespace by appending `?w=1`, it could be great if I could apply `&filter=apiv2|api:v2` in some form like I can with my native diff tools.